projects/widgets/docs/configureren/zoekresultaten.md

```diff
@@ -193,7 +193,7 @@ Je kan bepalen welke specifieke labels mogen worden getoond. Als je meerdere lab
 
 #### Toegankelijkheid
 
-Bepaal of de [toegankelijkheidsvoorzieningen](../../../uitdatabank/docs/terms.md#Facility) die aanwezig zijn op een evenement mogen worden getoond of niet.
+Bepaal of de [toegankelijkheidsvoorzieningen](../../../uitdatabank/docs/taxonomy-api/terms.md#Facility) die aanwezig zijn op een evenement mogen worden getoond of niet.
```

Comment:
Note: This was a quick fix for the dead link checker, but this relative link actually won't work on Stoplight because you cannot use relative links across projects. I will fix this later with an absolute link once the new URL of the terms guide is merged & published.